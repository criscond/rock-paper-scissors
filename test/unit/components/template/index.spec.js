import renderTheGame from '../../../../src/js/components/template';
describe('Template', () => {

  it('should create the template', () => {
    const template = renderTheGame();

    expect(template).toMatchSnapshot();
  });
});
