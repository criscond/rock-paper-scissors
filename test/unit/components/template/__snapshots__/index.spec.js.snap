// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template should create the template 1`] = `undefined`;
